packages/protect/src/index.ts

```diff
@@ -61,6 +61,7 @@ export const protect = async (
 export type { Result } from '@byteslice/result'
 export type { ProtectClient } from './ffi'
 export { csTable, csColumn } from './schema'
+export type { ProtectColumn, ProtectTable, ProtectTableColumn } from './schema'
```

Comment:
Heads up that there are some new exports here in Protect.js if you hadn't noticed already @calvinbrewer

Reply:
Yep all good! Added protect to the changeset